platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -390,7 +390,7 @@ private String getCertificateAlias() {
     return System.getProperty("org.codice.ddf.system.hostname");
   }
 
-  private KeyStore getSystemKeyStore() {
+  public KeyStore getSystemKeyStore() {
```

Comment:
❓ Is there a better way to get the system's keystore @stustison?

Reply:
AFAICT this is a viable & secure way to get the keystore. Since there are no `AccessController.doPrivileged` calls in here, a module would have to be given explicit access to the keystore files in the security manager policy files.

Just remember to wrap any calls to this method in `AccessController.doPrivileged`:
```
    AccessController.doPrivileged((PrivilegedAction<Keystore>) () -> getSystemKeystore());
```